core/start-service.sh

```diff
@@ -8,6 +8,9 @@ if [ ! -e /kopano/$SERVICE_TO_START.py ]
 then
     echo "Invalid service specified: $SERVICE_TO_START" | ts
     exit 1
+else
+	#ensure removed pid-file on unclean shutdowns and mounted volumes
+	rm -f /var/run/kopano/$SERVICE_TO_START.pid
```

Comment:
I did not try this yet, but the bash script will immediately exit on non zero return codes and if you put it into the else the command will always be executed (and definitely fail on a fresh start).

I would recommend to put it in its own if and check if the file exists before deleting it. Forcing rm also seems drastic .

Reply:
ok, then this change at least works as intended.

What do you think @zokradonh keep it in the else or make an own check for it? (its more of a style question, then)

Reply:
Maybe this is interesting for the discussion. I tried to keep it in sync with the style of the z-push apache pid file removal on startup:
https://github.com/zokradonh/kopano-docker/blob/87c814d491a88b442ee98d3100786a10fb111f74/zpush/start.sh#L43

Reply:
Huh, indeed.

For that case there is no variable involved, though. So what if:
```
SERVICE_TO_START="../../ "
rm -f /var/run/kopano/$SERVICE_TO_START.pid
```
But that is all hypothetical.

Reply:
> Ok i see your point. I think to get more robustness, we could check that the expression /var/run/kopano/$SERVICE_TO_START.pid points at least to a file with ending .pid in the desired folder.

I do not have much experience in writing non-exploitable bash checks. So if you want more checks here, please add those as you like.

Reply:
Yes, valid point about not having the -r flag. I think I have "gepiesackt" you for long enough. ;-) Lets keep the `rm -f` for now. I will though still make a small styling change after merging this.

Thanks for your contribution!